### ISSUE TITLE
Default user settings not applied

### ISSUE THREAD

**Issue by oseg** [iionly/elggchat#3]
In $enable_chat, $allow_contact_from & $show_offline_user default init, replace "==" by "=" (file views/default/forms/elggchat_usersettings/save.php)
```
<?php
$user = $vars['user'];
$enable_chat = elgg_get_plugin_user_setting('enableChat', $user->getGUID(), 'elggchat');
if (empty($enable_chat)) {
	$enable_chat = "yes";
}
$allow_contact_from = elgg_get_plugin_user_setting('allow_contact_from', $user->getGUID(), 'elggchat');
if (empty($allow_contact_from)) {
	$allow_contact_from = "friends";
}
$show_offline_user = elgg_get_plugin_user_setting('show_offline_user', $user->getGUID(), 'elggchat');
if (empty($show_offline_user)) {
	$show_offline_user = "no";
}

```

**Comment by iionly** [comment]
Thanks for reporting.

It's fixed in versions 1.10.8 and 2.0.2.

**Comment by oseg** [comment]
You're welcome and thank you for all these good plugins.
